desktop/core/src/desktop/lib/botserver/views_tests.py

```diff
@@ -47,4 +47,20 @@ def test_say_hi_user(self):
       }
       response = say_hi_user("channel", "user_id")
       api_call.assert_called_with(api_method='chat.postMessage', json={'channel': 'channel', 'text': 'Hi <@user_id> :wave:'})
-      assert_true(response['ok'])
\ No newline at end of file
+      assert_true(response['ok'])
+  
+  def test_handle_on_message(self):
+    with patch('desktop.lib.botserver.views.say_hi_user') as say_hi_user:
```

Comment:
Crux will be to do the same but for unfurling event ;)

Reply:
Added the unfurling event unit test

Reply:
Added the UT